dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/config/MasterConfig.java

```diff
@@ -74,6 +74,12 @@ public class MasterConfig implements Validator {
      */
     private String masterRegistryPath;
 
+    /**
+     * Maximum time allowed for a task to be successfully dispatched.
+     * Default: 5 minutes.
+     */
+    private Duration dispatchTimeout = Duration.ofMinutes(5);
```

Comment:
```suggestion
    private Duration maxTaskDispatchDuration;
```
`dispatchTimeout` might confuse with single RPC timeout when dispatch task, and the default value should be null or a large duration, should compatibility with the previous behavior.

And you need to add this in document.

Reply:
> maxTaskDispatchDuration

ok